src/main/resources/org/jenkinsci/plugins/scriptler/ScriptlerManagement/runScript.properties

```diff
@@ -22,7 +22,8 @@
 
 title=Run a script
 intro=Please select a slave you would like to run the script on. You can also modify the script to your needs, the changes will not effect the stored version of it!<br /><br />\
-You can also run a script using GET or POST to the URL /scriptler/run/&lt;your-script-id&gt; with the optional parameters node, script (alternative script text), contentType, and the parameters defined for your stored script.
+You can also run a script using POST to the URL <code>/scriptler/run/&lt;your-script-id&gt;</code> with the optional parameters <code>node</code>, <code>script</code> (alternative script text), <code>contentType</code>, and the parameters defined for your stored script.

```

Comment:
The "{{GET}} or {{POST}}" modification of the documentation is a correction of a modification done in another commit already merged. 

@imod the wiki page could also be changed when the 1.8 will be released in the REST chapter

This was corrected for SECURITY-334 by commit [50072e54c](https://github.com/jenkinsci/scriptler-plugin/commit/50072e54c443b9a894100e8771e492bddaf88a7d)

Reply:
👍 sorry, I did not find time to take a closer look at this PR yet - will try to find some on the next weekend

Reply:
@imod just a ping in case you forgot this PR